### DISCUSSION TITLE
Multiphase Flow Of Water Jet in the Air Simulation Using MOOSE.

### DISCUSSION THREAD

**Discussion by MashrurShejan** [idaholab/moose#23795] [Q&A General] [answered]
Hi everyone! I am trying to simulate multiphase flow of water jet in air using MOOSE to study the turbulence induced liquid sheet breakup process. I am pretty much new with using moose. Can anyone provide me a sample code or step by step guidelines to develop the input code? Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You may have a look at the Navier Stokes module to get started. 
We dont have any multiphase flow capability currently but the equations have some similarities so this will be a good example
https://mooseframework.inl.gov/modules/navier_stokes/index.html

Guillaume

**Reply by MashrurShejan** [reply]
Thank you for your valuable feedback. can you give me a basic structure of the code that I can follow?

**Reply by GiudGiud** [reply]
You should first follow the tutorials on how to use MOOSE

then you should determine whether you want to use finite element or finite volume
